.github/workflows/prerelease.yml

```
@@ -9,26 +9,20 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.9.7'
+  GO_VERSION: '1.16'
   INTEGRATION: "cassandra"
   ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
   REPO_FULL_NAME: ${{ github.event.repository.full_name }}
   TAG: ${{ github.event.release.tag_name }}
 
 jobs:
-
-  validate:
-    name: Validate code via linters
+  golangci-lint:
+    name: Validate code via golangci
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
-          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-      - name: Validate code
-        run: make ci/validate
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

Specify `only-new-issues` so CI does not fail for unrelated issues on PRs.
```suggestion
        uses: golangci/golangci-lint-action@v2
        with:
          only-new-issues: true

```

---

done in the PR!